doc/release-notes-26576.md

```diff
@@ -0,0 +1,5 @@
+Updated RPCs
+----------
+
+- RPC `disconnectnode` now accepts a subnet into `address`
```

Comment:
The release notes could be a bit more detailed, maybe mentioning that this aligns `DisconnectNode` and the `disconnectnode` rpc. It's nice to have at least a summary of the motivation in the release notes.

Reply:
>maybe mentioning that this aligns DisconnectNode and the disconnectnode rpc. 

Internal APIs really don't seem like they belong here

Reply:
I don't think it's worth to mention this in a release note, seems like a specific code nuance to be exposed. Isn't it?